scripts/babel-relay-plugin/src/GraphQLPrinter.js

```diff
@@ -499,9 +506,6 @@ function printField(
         ));
       }
     });
-    if (hasEdgesSelection && !!fieldDecl.type.getFields()['pageInfo']) {
-      subRequisiteFields.pageInfo = true;
-    }
   } else if (types.getNamedType(fieldDecl.type).name === 'PageInfo') {
```

Comment:
Not a blocker for this PR, but I wonder if these other checks (that add `subRequisiteFields`) should also be moved up a level.